client/core/core.go

```diff
@@ -218,6 +218,51 @@ func (dc *dexConnection) setRegConfirms(confs uint32) {
 	dc.regConfirms = &confs
 }
 
+func (dc *dexConnection) exchangeInfo() *Exchange {
+	// Set AcctID to empty string if not registered.
+	acctID := dc.acct.ID().String()
+	var emptyAcctID account.AccountID
+	if dc.acct.ID() == emptyAcctID {
+		acctID = ""
+	}
+
+	dc.cfgMtx.RLock()
+	cfg := dc.cfg
+	dc.cfgMtx.RUnlock()
+	if cfg == nil { // on config, assets, or markets data
+		return &Exchange{
+			Host:       dc.acct.host,
+			AcctID:     acctID,
+			FeePending: dc.acct.feePending(),
+			Connected:  atomic.LoadUint32(&dc.connected) == 1,
+			// RegConfirms:   dc.getRegConfirms(), // could show, but we don't know required
+		}
+	}
+
+	dc.assetsMtx.RLock()
+	assets := make(map[uint32]*dex.Asset, len(dc.assets))
+	for assetID, dexAsset := range dc.assets {
+		assets[assetID] = dexAsset
+	}
+	dc.assetsMtx.RUnlock()
+
+	return &Exchange{
+		Host:          dc.acct.host,
+		AcctID:        acctID,
+		Markets:       dc.marketMap(),
+		Assets:        assets,
+		FeePending:    dc.acct.feePending(),
+		Connected:     atomic.LoadUint32(&dc.connected) == 1,
+		ConfsRequired: uint32(cfg.RegFeeConfirms),
+		RegConfirms:   dc.getRegConfirms(),
+		Fee: &FeeAsset{ // v0 is only DCR, but there will be a RegFees map[string]*FeeAsset field
+			ID:    42,
+			Amt:   cfg.Fee,
+			Confs: uint32(cfg.RegFeeConfirms),
+		},
```

Comment:
In anticipation of multiple registration fee assets, which is started in https://github.com/decred/dcrdex/pull/1017/files#diff-6fe96257ee07551d0c31a6e6d14c4d9557b48de4262b5fede53cc307eaee5a91R1100